solutions/platinum/leetcode-1453.mdx

```diff
@@ -0,0 +1,108 @@
+---
+id: leetcode-1453
+source: Leetcode
+title: Maximum Number of Darts Inside of a Circular Dartboard
+author: Mihnea Brebenel
+---
+
+## Explanation
+
+The general idea is the following: we can try finding the answer
+by placing the circle's boundary on each point, then sweep - or rotate -
+the circle around the point to see the maximum number of points that can be inside the circle.
+The picture below depicts how the sweep works:
+
+![angular-sweep](./leetcode-1453/angular-sweep.png)
+
+The sweeping around point $p$ can be done in $\mathcal{O}(N \cdot \log{N})$ by
+computing for each point the angle of entering and exiting the circle. Afterward
+traverse the sorted array of angles while keeping track of active points i.e.
+points lying inside the circle. The considered angles will be relative
+to the center of the circle. Let's analyze how to find the entrace angle for
+a point:
+
+![angular-sweep](./leetcode-1453/angular-sweep-enter.png)
+<i><center>$O$ - circle's center; $Q$- arbitrary chosen point; $r$ - radius of circle; $a, b, \alpha$ - angles</center></i>
```

Comment:
well it isn't really an arbitrary chosen point right
it's a point we wanna test for inside-circle-ness

Reply:
yes

chosen from the given list of points

Reply:
make that more explicit

Reply:
alright